src/main/webapp/css/level5.css

```diff
@@ -0,0 +1,5 @@
+#map {
```

Comment:
did the css thing fix it?

Reply:
um I changed the css and it didnt work, so I changed it to async defer and it worked but there was an error so I tired it on my map html without async and it worked. I took the async out and back to how I normally had it but with the new css and it worked so I guess the css was the problem but idk haha